src/components/customerInfo/CustomerInfo.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Radio } from "@patternfly/react-core";
+import { Button } from "@patternfly/react-core";
+import "./CustomerInfo.scss";
+import { Link } from "react-router-dom";
+
+function CustomerInfo() {
+  return (
+    <div className="customer-info-main-container">
+      <div className="customer-info-top-container">
+        <div className="radio-contianer">
+          <Radio
+            id="radio-standalone"
+            aria-label="Standalone input"
+            name="exampleRadioStandalone"
+            isChecked="true"
+          />
```

Comment:
name for this component could be better

Reply:
Radio is a patternfly component                                                                                                                                        import { Radio } from "@patternfly/react-core";

Reply:
Radio is a pattern fly component hence the component name cannot be changed.